lighthouse-core/scripts/update-report-fixtures.js

```diff
@@ -7,6 +7,8 @@
 
 const cli = require('../../lighthouse-cli/run.js');
 const cliFlags = require('../../lighthouse-cli/cli-flags.js');
+const assetSaver = require('../lib/asset-saver');
```

Comment:
```suggestion
const assetSaver = require('../lib/asset-saver.js');
```

Reply:
Can we have a lint rule for this? And update all the places where we don't use `.js` right now?

Reply:
> Can we have a lint rule for this? 

I'd love that. I couldn't find a good one last time I looked, but I didn't try very hard to find one.